examples/failure_and_success_hooks/hooks/success_hooks.py

```diff
@@ -0,0 +1,19 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from zenml.steps import StepContext
+
+
+def pipeline_success_hook(context: StepContext) -> None:
+    print(f"Step {context.step_name} succeeded!")
```

Comment:
Should this maybe be logging instead of printing? Maybe encouraging antipatterns somehow?

Reply:
Makes sense!